service/src/main/docker/README.txt

```diff
@@ -0,0 +1,16 @@
+A: Build RemRem-Generate Docker image based on RemRem-Generate artifact from an Artifactory, e.g. Jitpack:
+cd (git root dir)/service
+docker build -t remrem-generate:0.10.8 --build-arg URL=https://jitpack.io/com/github/Ericsson/eiffel-remrem-generate/generate-service/0.10.8/generate-service-0.10.8.war -f src/main/docker/Dockerfile .
```

Comment:
Is the url correct with Ericsson in it? Shouldn't it be eiffel-community?

Reply:
Fixed.